bin/add-author.js

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env node
+const minimist = require('minimist');
+const { gitAuthors } = require('../src/git-authors');
+const { runAddCoauthorHelp, validateEmail } = require('../src/helpers');
+
+const argv = minimist(process.argv.slice(2), {
+  alias: {
+    h: 'help',
+  },
+});
+
+async function addCoauthor(argv) {
+  if (argv.help) {
+    runAddCoauthorHelp();
+    process.exit(0);
+  }
+  const instance = gitAuthors();
+  const authorList = await instance.read();
+  const args = argv._;
+
+  if (args.length !== 3) {
+    console.log('Incorrect Number of Parameters');
+    process.exit(0);
```

Comment:
These look like they should be errors: Change to `console.error` and `exit` code should be `1`.